### ISSUE TITLE
Backfill label table with timestamp of the creation date/time

### ISSUE THREAD

**Issue by manaswis** [ProjectSidewalk/SidewalkWebpage#974]
Based on a recent issue #967, we decided to add a new column in the label table to store timestamps for label creation date/time - https://github.com/ProjectSidewalk/SidewalkWebpage/issues/967#issuecomment-321855920.

So this requires us to backfill the timestamps for the old data that's already stored.

**Comment by misaugstad** [comment]
I actually ended up doing this as part of #3033. It was easier to to just guarantee that the timestamp column is always filled over dealing with the possibilities of nulls, so I went ahead and backfilled the table as part of that PR.

However, I forgot to actually make the column non-null. It's impossible for us to ever add a null value into the database now through the code, but I still need to actually set that flag in the database. I'll close this ticket once I do that.
